### ISSUE TITLE
Can't scrape games with - (dash) in them?

### ISSUE THREAD

**Issue by Jaxel** [Universal-Rom-Tools/Universal-XML-Scraper#218]
I can't seem to scrape information for games with dashes in them... games like X-Men or Banjo-Kazooie.

I also can't seem to scrape any MAME/FBA games.

**Comment by Universal-Rom-Tools** [comment]
Not sure it come from the dash :s 

UXS check matching game with HASH at first (so no problem with filename)
If not found, it try to match with the filename. In this case, the filename MUST be the same as one on the DB. (Regular romset are nearly all in the DB... so if you rename them, it won't work...)

for Mame/FBA, it's strange. Rom filename are standard and normalized... So even if you have a strange romset with unknown HASH, it will find game with their filename...

You can check yourself going to a game in www.screenscraper.fr and look for referenced filename at the very end of the web page.

**Comment by Jaxel** [comment]
Yeah, I have the right ROM names... hashes won't match though because I merged them... but the zip files do have the correct names.
